src/api/common/HttpWrapper.ts

```diff
@@ -50,21 +50,23 @@ const interceptors = {
     onError: (error: AxiosError) => {
       const { response, message, config } = error;
       if (error.request) {
-        log.error(`Request failed: ${stringify(error.request)}`);
+        logger().error(`Request failed: ${stringify(error.request)}`);
       }
       if (response) {
-        log.error(`Response error: ${stringify(response)}`);
+        logger().error(`Response error: ${stringify(response)}`);
       }
       if (!error.request && !response) {
-        log.error('Error while setting up request', stringify(message));
+        logger().error('Error while setting up request', stringify(message));
       }
-      log.debug(`Configuration: ${stringify(config)}`);
+      logger().debug(`Configuration: ${stringify(config)}`);
 
       return Promise.reject(error);
     },
   },
 };
 
+const logger = () => log.getLogger('HttpWrapper');
```

Comment:
I know it's the same, but I'd prefer to define the `logger` function at the top of the file.

Reply:
updated